Primo.js

```diff
@@ -265,8 +278,10 @@ function importPNX(text) {
 	}
 	
 	item.title = ZU.xpathText(doc, '//display/title');
-	if(item.title) item.title = ZU.unescapeHTML(item.title);
-	
+	if(item.title) {
+		item.title = ZU.unescapeHTML(item.title);
+	    item.title = item.title.replace(/\s*:/, ":");
```

Comment:
Yeah, I was just wondering... BTW I think the space-semicolon-space comes from the [ISBD](https://de.wikipedia.org/wiki/International_Standard_Bibliographic_Description) format.